<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetherPosClass</class>
 <widget class="QMainWindow" name="NetherPosClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>401</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>NetherPos</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="radioGroupBox">
      <property name="title">
       <string>Mode</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="netherRadioButton">
         <property name="text">
          <string>Get Nether Position</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="worldRadioButton">
         <property name="text">
          <string>Get World Position</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="instructionLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="xLabel">
        <property name="text">
         <string>X :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="xSpinBox">
        <property name="minimum">
         <number>-30000000</number>
        </property>
        <property name="maximum">
         <number>30000000</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="zLabel">
        <property name="text">
         <string>Z:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="zSpinBox">
        <property name="minimum">
         <number>-30000000</number>
        </property>
        <property name="maximum">
         <number>30000000</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="calculatePushButton">
      <property name="text">
       <string>Calculate</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="positionResultLabel">
        <property name="text">
         <string>Position:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="resultLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>354</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="NetherPos.qrc"/>
 </resources>
 <connections/>
</ui>
